app/views/posts/_post.html.erb

```diff
@@ -0,0 +1,13 @@
+<% @topic.posts.each do |post| %>
```

Comment:
Ok, that actually makes a lot of sense. I updated the code to reflect the changes you suggested.